mws/mws.py

```diff
@@ -307,6 +310,7 @@ def make_request(self, extra_data, method="GET", **kwargs):
             # to convert the dict to a url parsed string, so why do it twice if i can just pass the full url :).
             response = request(method, url, data=kwargs.get(
                 'body', ''), headers=headers, proxies=proxies, timeout=kwargs.get('timeout', 300))
+            response.encoding = 'utf-8'
```

Comment:
Is this overwrite wise? The encoding from MWS is ISO 8859-1 by default ([docs](http://docs.developer.amazonservices.com/en_US/dev_guide/DG_ISO8859.html))

Reply:
Yep, thats my worry is we break a lot more things, the law of unintended consequences etc!